### PR TITLE
Fix BankAccount SwiftCode

### PR DESCRIPTION
**Pull request by riadmen**
https://github.com/metasfresh/metasfresh/issues/12817

### CODE REVIEW

File: backend/de.metas.business/src/main/java-legacy/org/compiere/model/MPayment.java [hunk 1]
```diff
@@ -628,9 +628,13 @@ && isReceipt())
...
 			{
+				if (getIBAN() != null)
+				{
+					documentNo = getIBAN() + ": ";
+				}
 				if (getRoutingNo() != null)
 				{
-					documentNo = getRoutingNo() + ": ";
+					documentNo += getRoutingNo();
 				}
```
Comment (cristinamghita) [inline backend/de.metas.business/src/main/java-legacy/org/compiere/model/MPayment.java:638]: I don;t understand this.

File: backend/de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerBankAccountImportHelper.java [hunk 2]
```diff
@@ -79,7 +79,7 @@ else if (!Check.isEmpty(importRecord.getSwiftCode(), true) && !Check.isEmpty(imp
 			bankAccount = InterfaceWrapperHelper.newInstance(I_C_BP_BankAccount.class);
 			bankAccount.setC_BPartner_ID(bpartnerId.getRepoId());
 			bankAccount.setIBAN(importRecord.getIBAN());
-			bankAccount.setA_Name(importRecord.getSwiftCode());
+			bankAccount.setSwiftCode(importRecord.getSwiftCode());
```
Comment (cristinamghita) [inline backend/de.metas.business/src/main/java/de/metas/bpartner/impexp/BPartnerBankAccountImportHelper.java:82]: Put a name as well, please.
You can still use swift code as a name or generate a combination.

**Review by cristinamghita — COMMENTED** [review]
Check my comments

File: backend/de.metas.payment.revolut/src/main/java/de/metas/payment/revolut/model/RevolutExportCSVColumn.java [hunk 3]
```diff
@@ -34,7 +34,7 @@
 	AccountNo("Account number"),
 	RoutingNo("Sort code or Routing number"),
 	IBAN("IBAN"),
-	BIC("BIC"),
```
Comment (metas-ts) [inline backend/de.metas.payment.revolut/src/main/java/de/metas/payment/revolut/model/RevolutExportCSVColumn.java:37]: AFAIU
- BIC is the name of the column in the revolut standard.
- BIC is practically the same as switftcode

=> therefore, please leave the name as BIC, at least in the csv output file

**Review by metas-ts — CHANGES_REQUESTED** [review]

**Review by metas-ts — APPROVED** [review]

**Pull request merged**